lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -299,7 +301,15 @@ public function update_item( $request ) {
 
 		$this->update_additional_fields_for_object( $user, $request );
 
-		do_action( 'rest_insert_user', $user, $request, false );
+		/**
+		 * Fires after a user is updated via the REST API.
+		 *
+		 * @param object          $user    Data used to update the user (not a WP_User object).
+		 * @param WP_REST_Request $request Request object.
+		 * @param bool            $bool    A boolean that is false.
+		 * @todo remove final bool = false variable, unused?
+		 */
+		do_action( 'rest_update_user', $user, $request, false );
```

Comment:
@adamsilverstein to keep the same naming convention in the other endpoint controllers, it should be `rest_insert_user` but with the third param of `false`.